config/initializers/assets.rb

```diff
@@ -102,4 +102,7 @@
       ActionView::Base.assets_manifest = app.assets_manifest
     end
   end
+  #Absulote static URL for mailer
+  #url = URI.parse(Setting[:foreman_url])
+  #qqconfig.roadie.url_options = {:host => url.host, :port => url.port, :protocol => url.scheme}
```

Comment:
Well, or should it be moved from development.rb so it applies to all environments?

What I mean to say above is - there shouldn't be commented out code in a proposed patch.